setup.py

```diff
@@ -14,7 +14,7 @@
     license         = 'BSD',
     platforms       = "Linux, Mac OS X, Windows",
     keywords        = ['Jupyter', 'JupyterLab'],
-    python_requires = '>=3.6',
+    python_requires = '>=3.5',
```

Comment:
I've come across this on another Git extension as well - Is there somewhere Jupyter mandates Py3.5 compatibility?

Reply:
I added it to make https://github.com/jupyter/notebook/pull/4752 pass, since notebook's tests seem to require a minimum of python 3.5.